Marlin/Conditionals_post.h

```diff
@@ -780,11 +780,11 @@
   #endif
 
   #if IS_KINEMATIC
-    // Check for this in the code instead
-    #define MIN_PROBE_X X_MIN_POS
-    #define MAX_PROBE_X X_MAX_POS
-    #define MIN_PROBE_Y Y_MIN_POS
-    #define MAX_PROBE_Y Y_MAX_POS
+    // Check for this in the code instead (preprocessor can't do floating point arithmetic)
+    #define MIN_PROBE_X (-10000)
+    #define MAX_PROBE_X ( 10000)
+    #define MIN_PROBE_Y (-10000)
+    #define MAX_PROBE_Y ( 10000)
```

Comment:
SCARA with UBL still has problems unrelated to my changes.  UBL requires NEWPANEL which the SCARA default configuration.h does not provide (LCD errors when building with UBL).  

So for now UBL+SCARA disabled in sanitycheck.h but all the functionality for SCARA should be in UBL_DELTA, but unable to build to verify.